cob_twist_controller/src/cob_twist_controller.cpp

```diff
@@ -187,6 +187,7 @@ void CobTwistController::reconfigure_callback(cob_twist_controller::TwistControl
     params.limits_min = twistControllerParams_.limits_min; // from cob_twist_controller init
     params.limits_max = twistControllerParams_.limits_max; // from cob_twist_controller init
     params.p_gain = config.p_gain;
+    params.beta = config.beta;
```

Comment:
same here: move beta up (under wt) to be consistent with the order in the cfg-file...